extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -98,6 +98,10 @@ protected function requestParsoid( $method, $title, $params ) {
 			}
 		} elseif ( $status->isGood() ) {
 			$this->dieUsage( $req->getContent(), 'parsoidserver-http-' . $req->getStatus() );
+			\Wikia\Logger\WikiaLogger::instance()->info( 'ApiVisualEditor_requestParsoid', [
```

Comment:
You're reporting an error here, right? Please use `error` level.

This will allow us to automatically report an error spike.